aggregator/src/aggregator/aggregate_init_tests.rs

```diff
@@ -487,6 +489,80 @@ async fn aggregation_job_mutation_report_aggregations() {
     assert_eq!(response.status(), Some(Status::Conflict));
 }
 
+#[tokio::test]
+async fn aggregation_job_intolerable_clock_skew() {
+    let mut test_case = setup_aggregate_init_test_without_sending_request(
+        dummy_vdaf::Vdaf::new(),
+        VdafInstance::Fake,
+        dummy_vdaf::AggregationParam(0),
+        (),
+        AuthenticationToken::Bearer(random()),
+    )
+    .await;
+
+    test_case.aggregation_job_init_req = AggregationJobInitializeReq::new(
+        test_case.aggregation_param.get_encoded().unwrap(),
```

Comment:
Note that this needs some rebase cleanup to roll back to infallible Encode.

Reply:
D'oh, thanks!